dxp/aui_script.md

```diff
@@ -0,0 +1,148 @@
+
+#### Import a ES module using `require` property
+
+Allows users to require a modern ES module as in the following example:
+
+Considering We already have `metal-clipboard` and `metal-dom` dependencies listed [on our module package.json](https://help.liferay.com/hc/en-us/articles/360018159771-The-Structure-of-OSGi-Bundles-Containing-npm-Packages-)
+
+By default the name of the loaded resource will be the transformed path to camel-case default exported module/file:
+
+```jsp
+<aui:script require="metal-clipboard/src/Clipboard">
+    // Note We have the provided path to require metal-clipboard/src/Clipboard normalized and camel-case as a variable name
+	new metalClipboardSrcClipboard.default();
+</aui:script>
+```
+
+Also, We can set an alias with the `as` word after the provided path. It will be look like:
+
+```jsp
+<aui:script require="my-custom-lib/src/path/main as myLib">
+	// ... code that use some function from myLib like `myLib.someFunction();`
+</aui:script>
+```
+
+The output will be:
+
+```js
+Liferay.Loader.require('my-custom-lib/src/path/main', function(myCustomLibSrcPathMain) {
+	try {
+		(function() {
+			var myLib = myCustomLibSrcPathMain;
+			var $ = AUI.$;var _ = AUI._;
+			// ... code that use some function from myLib like `myLib.someFunction();`
+		})();
+	} catch (err) {
+		console.error(err);
+	}
+}
+```
+
+You can find more information on [this link](https://help.liferay.com/hc/en-us/articles/360017882752-Loading-Modules-with-AUI-Script-in-Liferay-DXP#loading-es2015-and-metaljs-modules)
+
+#### Using `position` to control script placement
+
+This property will change the way in which / how the script will be injected into the page. By default, those scripts that are wrapped by `aui:script` will be added to the minified files as described [here](#inline-js-on-the-jsp).
+
+When placing the option of `inline` for this property, the scripts will be placed in a script tag obeying the order of the DOM where the JSP is being rendered. See [here](https://github.com/liferay/liferay-portal/blob/815f48f484351e18b61e4b9c9fbf40f0609bdc56/util-taglib/src/com/liferay/taglib/aui/ScriptTag.java#L143..L165).
+
+#### `sandbox`
+
+The purpose for the sandbox attribute is to wrap the contents of the script tag in an [Immediately Invoked Function Expression/IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).
+
+As a short brief, it will limit the scope of the script tag. See the following example:
+
+```jsp
+<aui:script>
+   var myVar = 'foo'; // This variable will be available globally
+</aui:script>
+
+<aui:script sandbox="<%= true %>>
+   var myVar = 'foo'; // This variable will NOT be available globally
+</aui:script>
+```
+
+#### Caveats
+
+You can't use both require and use in the same tag. use is for declaring dependencies on AUI modules, while require gives access to ES modules.
```

Comment:
require -> `require`
use -> `use`

Note that each term appears twice ;-).

Reply:
Note that each term appears twice ;-).

Thanks for point up! :)